packages/web3torrent/src/constants.ts

```diff
@@ -65,37 +65,24 @@ export const EmptyTorrent = ({
   wires: []
 } as unknown) as Torrent;
 
-// Mocked Constants
-export const mockTorrents: Array<Partial<Torrent>> = [
+// Pre Seeded Constants (by StateChannels team)
+export const preSeededTorrents: Array<Partial<Torrent>> = [
   {
-    name: 'Sintel',
-    length: 129302391,
-    numSeeds: 47,
-    numPeers: 12,
-    files: [],
+    name: 'Sintel.mp4',
+    length: 129241752,
+    infoHash: 'c53da4fa28aa2edc1faa91861cce38527414d874',
     magnetURI:
-      'magnet:?xt=urn:btih:08ada5a7a6183aae1e09d831df6748d566095a10&dn=Sintel&tr=udp%3A%2F%2Fexplodie.org%3A6969&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Ftracker.empire-js.us%3A1337&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2F&xs=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel.torrent'
```

Comment:
Here lies the only remains of any public trackers. This is the reason we were still trying to connect to the defunct `tracker.fastcast.nz` despite removing it from our list of trackers.